docs/rfcs/0001-rfc-process.md

```diff
@@ -0,0 +1,70 @@
+- Status: Proposed
+- Date: 2023-05-29
+- Author: [@lesleyjanenorton](https://github.com/lesleyjanenorton)
+- RFC PR: [#6981](https://github.com/mozilla-mobile/mozilla-vpn-client/pull/6982)
+- Implementation GitHub issue: Not applicable. Merging [#6981](https://github.com/mozilla-mobile/mozilla-vpn-client/pull/6982) implements the proposal.
+
+
+## Summary
+Introducing a process for proposing, reviewing, and documenting decisions with high-impact implications for our codebase.
+
+## Motivation
+
+RFCs authored by this team are currently submitted and reviewed in a number of places, namely Confluence and Google Docs.
+
+Several challenges have been noted with this practice:
+- Docs are accessible to Mozillians only
+- Platform constraints make asynchronous discussions difficult and impossible to reference later (Confluence) 
+- Uneven awareness and ability for interested parties to participate in decisions as they're being made (Google Docs) 
+
+We need a more inclusive, more collaborative process for making decisions that impact our codebase.
+
+## Proposed Solution
+
+RFCs will be submitted and reviewed as pull requests in this repository. If approved, they'll be stored as documentation.
+
+**Salient points in favor**
+ - Viewable by all, not just Mozilla-internals
+ - Closer to the code, easier to find
+ - In keeping with how other teams do it (TODO link evidence)
+
+
+## Guide-level explanation
+
+The high-level process to create an RFC is as follows:
+
+* Create an RFC document (like this one, in Markdown).
+* Open a pull request for the RFC document and prefix the title of the PR with `[RFC]`.
+* File an issue requesting feedback on the pull request.
```

Comment:
This feels backwards. Filing the ticket shold be the first step. Especially for people inside the team, given this will need to be added to our sprints during spring planning. Also, it's worth specifying "file a public ticket".

Reply:
Aren't all tickets filed in Jira copied over to GitHub unless explicitly given the 'no-sync' label? I'm fine with adding this stipulation regardless.